File: dataset/human/java/file_5532.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;


public class Main {
	public static void main(String[] args) throws IOException{


		// 読み込み
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String line;
		String[] temp;

		line=br.readLine();
		temp=line.split(" ");

		int x = Integer.valueOf(temp[0]);

		if(x <1200){
			System.out.println("ABC");
		}else{
			System.out.println("ARC");
		}


	}
}

File: dataset/human/python/file_5532.py
X,Y = (str(i) for i in input().split())

def odd16(Z):
    if Z == 'A':
        return(11)
    elif Z == 'B':
        return(12)
    elif Z == 'C':
        return(13)
    elif Z == 'D':
        return(14)
    elif Z == 'E':
        return(15)
    elif Z == 'F':
        return(16)
    
x = odd16(X)
y = odd16(Y)

if x == y:
    print('=')
elif x > y:
    print('>')
else:
    print('<')
